luigi_pipeline/lib/model/gcnv_mt_schema.py

```diff
@@ -110,11 +110,13 @@ def sorted_transcript_consequences(self):
         sorted_transcript_consequences,
     ])
     def transcript_consequence_terms(self):
-        default_consequences = hl.set([hl.format('gCNV_%s', self.mt.svType)])
-        gene_major_consequences = hl.set(self.mt.sortedTranscriptConsequences.map(lambda x: x[MAJOR_CONSEQUENCE]))
-        return hl.array(
-            default_consequences.union(gene_major_consequences)
-        )
+        default_consequences = [hl.format('gCNV_%s', self.mt.svType)]
+        gene_major_consequences = hl.array(hl.set(
+            self.mt.sortedTranscriptConsequences
+            .filter(lambda x: x.contains(MAJOR_CONSEQUENCE))
+            .map(lambda x: x[MAJOR_CONSEQUENCE])
+        ))
+        return default_consequences.extend(gene_major_consequences)
```

Comment:
do you need to explicitly cast `default_consequences` as a `hl.array`?

Reply:
yep! or just flip the ordering 😬!